website/i18n/zh/docusaurus-plugin-content-blog/2022/01/11/apisix-with-shardingsphere-meetup.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "直播预告 | 从网关到数据，分布式全链路在线应用实践"
+keywords: 
+- Apache APISIX
+- Apache ShardingSphere
+- Apache Skywalking
+- 数据库
+- CyborgFlow
+description: 1 月 15 日，Apache APISIX 社区将联合 Apache Shardingshpere 社区为大家带来线上分享，快来看看有哪些精彩的议题吧。
+tags: [Events]
+---
+
+> 1 月 15 日，Apache APISIX 社区将联合 Apache Shardingshpere 社区为大家带来线上分享，快来看看有哪些精彩的议题吧。
+
+<!--truncate-->
+
+![活动海报]!(https://static.apiseven.com/202108/1641870767554-2f443190-e752-48e5-9624-ca8ee0aca1e4.png)
```

Comment:
See https://deploy-preview-838--apache-apisix.netlify.app/zh/blog/2022/01/11/apisix-with-shardingsphere-meetup/ , the extra `!` behind makes the image fail.
```suggestion
![活动海报](https://static.apiseven.com/202108/1641870767554-2f443190-e752-48e5-9624-ca8ee0aca1e4.png)
```

Reply:
![image](https://user-images.githubusercontent.com/36651058/148907397-9f8e31ac-1460-442b-b24e-db51f1054e44.png)